dace/data.py

```diff
@@ -1680,6 +1680,35 @@ def set_shape(
         self._set_shape_dependent_properties(new_shape, strides, total_size, offset)
         self.validate()
 
+    def _get_packed_fortran_strides(self) -> Tuple[int]:
+        """Compute packed strides, if the array is stored Fortran-style (column-major)."""
+        accum = 1
+        strides = []
+        for shape in self.shape:
```

Comment:
Just noting that you can also write this with something like `return (1, ) + tuple(np.multiply.accumulate(shape[:-1]))`. It should be faster, but unsure if it would be worth the (potential?) decrease in readability.

Reply:
I am also wondering whether we should cache those strides instead of recomputing them.